src/main/java/com/andreamazzarella/contact_manager/ContactBuilder.java

```diff
@@ -1,11 +1,13 @@
 package com.andreamazzarella.contact_manager;
 
+import java.util.Optional;
+
 public class ContactBuilder {
 
     private String firstName = "";
     private String lastName = "";
     private String streetAddress = "";
-    private String postalCode = "";
+    private Optional<String> postalCode = Optional.of("");
```

Comment:
The empty string is an actual postcode now, shouldn't this be `Optional.empty()` to initialize it with "no value".